fashion-messaging-app-api/server.js

```diff
@@ -152,6 +167,58 @@ app.get('/photos', async (req, res) => {
   }
 });
 
+app.get('/recommendations', async (req, res) => {
+  try {
+    if (!req.session.user) {
+      return res.status(401).json({ error: 'Unauthorized' });
+    }
+
+    const lastSearch = await SearchResult.findOne({
+      where: { userId: req.session.user.id },
+      order: [['createdAt', 'DESC']],
+    });
```

Comment:
This checks each row in the database then returns the most recent searchResult that was stored